draft-ietf-scitt-architecture.md

```diff
@@ -895,78 +897,65 @@ By analyzing the relationship between data stored in the Transparency Service an
+- carefully protect their private signing keys
+- avoid using keys for more than one purpose
+- rotate their keys in well-defined cryptoperiods, see {{KEY-MANAGEMENT}}
 
-The Transparency Service may be replicated with a consensus algorithm, such as Practical Byzantine Fault Tolerance {{PBFT}} and may be used to protect against malicious or vulnerable replicas.
-Threshold signatures may be use to protect the service key, etc.
+Issuers and Transparency Services SHOULD:
 
-Issuers and Transparency Services MUST rotate their keys in well-defined cryptoperiods, see {{KEY-MANAGEMENT}}.
+- use a Trusted Execution Environment (TEE) for cryptographic operations.
 
-A Transparency Service MAY provide additional authenticity assurances about its secure implementation and operation, enabling remote attestation of the hardware platforms and/or software Trusted Computing Bases (TCB) that run the Transparency Service.
-If present, these additional authenticity assurances MUST be registered in the Verifiable Data Structure and MUST always be exposed by the Transparency Services' APIs.
-An example of Signed Statement's payloads that can improve authenticity assurances are trustworthiness assessments that are RATS Conceptual Messages, such as Evidence, Endorsements, or corresponding Attestation Results (see {{-rats-arch}}).
+## Transparency Service Operational Considerations
 
-For example, if a Transparency Service is implemented using a set of redundant replicas, each running within its own hardware-protected trusted execution environments (TEEs), then each replica can provide fresh Evidence or fresh Attestation Results about its TEEs.
-The respective Evidence can show, for example, the binding of the hardware platform to the software that runs the Transparency Service, the long-term public key of the service, or the key used by the replica for signing Receipts.
-The respective Attestation Result, for example, can show that the remote attestation Evidence was appraised by a Relying Party and complies with well-known Reference Values and Endorsements.
+Issuers and Transparency Services SHOULD:
 
-Auditors should be aware that the certification path information included in an unprotected `x5chain` header of a to-be-registered Signed Statement can be tampered with by a malicious Transparency Service (e.g., one that does not incorporate remote attestation), which may replace the intermediate certificates and ultimately connect to an unexpected root.
-This modification helps protect against person-in-the-middle attacks, but not denial-of-service.
-Auditors MUST perform certification path validation in accordance with PKIX rules specified in {{-PKIX}}.
-Auditors MUST verify that certification paths chain to one or more trust anchors (often represented as root certificates).
+- protect against malicious or vulnerable replicas by replicating with a consensus algorithm, such as Practical Byzantine Fault Tolerance {{PBFT}}.
```

Comment:
Why have we added back in specifics on replication, and in particular consensus algorithms ? There is no problem with vulnerable replicas provided an effective use of consistency proofs is used to ensure no split view.

Reply:
I don’t think these have been added back, they have been floating around for a while. There is no PBFT implementation, this text is not helpful for interoperability or for implementers.

Separately, I think there is value in hardening the TS, but I am sure that’s not controversial :)